inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -376,6 +376,49 @@ void FrontEnd::parseLayer(const Model& model, const ie::CNNLayerPtr& layer, cons
     }
 }
 
+void FrontEnd::processTrivialCases(const Model& model) {
+    std::map<ie::DataPtr, DataVector> ieToVpuDataVector;
+    std::for_each(model->datas().begin(), model->datas().end(), [&ieToVpuDataVector](const Data& data) {
```

Comment:
Nitpick: you call `model->datas` twice

Reply:
Nitpick: it looks to me plain for-each loop would look a bit easier

Reply:
Changed to plain `for` loop.
Thanks!